src/erf.jl

```diff
@@ -96,10 +97,211 @@ See also:
+
+
+        # Obtains erfc of |x|
+        r=evalpoly(a,PD)
+
+
+        if (sign)
+            return -1.0 + r
+        else
+            return 1.0 - r
+        end
+    elseif (ia < 0x40100000)
+    ## 3.25 <= |x| < 4.0.  
+        a = abs(x)
+        a = a - 3.25
+
+        # Generated using Sollya::remez(f(c*x+d), deg, [(a-d)/c;(b-d)/c], 1, 1e-16), [|D ...|] with deg=13 a=3.25 b=4 c=1 d=3.25 
+        PE=( 0x1.20c13035539e4p-18, -0x1.e9b5e8d16df7ep-16, 0x1.8de3cd4733bf9p-14, -0x1.9aa48beb8382fp-13, 0x1.2c7d713370a9fp-12, -0x1.490b12110b9e2p-12, 0x1.1459c5d989d23p-12, -0x1.64b28e9f1269p-13, 0x1.57c76d9d05cf8p-14, -0x1.bf271d9951cf8p-16, 0x1.db7ea4d4535c9p-19, 0x1.91c2e102d5e49p-20, -0x1.e9f0826c2149ep-21, 0x1.60eebaea236e1p-23 )
+
+        r=evalpoly(a,PE)
+
+
+        if (sign)
+            return -1.0 + r
+        else
+            return 1.0 - r
+        end
+    elseif (ia < 0x4017a000)
+    ## 4 <= |x| < 5.90625.  
+        a = abs(x)
+        a = fma(0.5, a, -2.0)
+
+        # Generated using Sollya::remez(f(c*x+d), deg, [(a-d)/c;(b-d)/c], 1, 1e-16), [|D ...|] with deg=16 a=4 b=5.90625 c=2 d=4 
+        PF=( 0x1.08ddd130d1fa6p-26, -0x1.10b146f59ff06p-22, 0x1.10b135328b7b2p-19, -0x1.6039988e7575fp-17, 0x1.497d365e19367p-15, -0x1.da48d9afac83ep-14, 0x1.1024c9b1fbb48p-12, -0x1.fc962e7066272p-12, 0x1.87297282d4651p-11, -0x1.f057b255f8c59p-11, 0x1.0228d0eee063p-10, -0x1.b1b21b84ec41cp-11, 0x1.1ead8ae9e1253p-11, -0x1.1e708fba37fccp-12, 0x1.9559363991edap-14, -0x1.68c827b783d9cp-16, 0x1.2ec4adeccf4a2p-19 )
+
+        r=evalpoly(a,PF)
+
+
+        if (sign)
+            return -1.0 + r
+        else
+            return 1.0 - r
+        end
+    else
+        
+        if(isnan(x))
+             return NaN
+        end
+
+        if (sign)
+            return -1.0
+        else
+            return 1.0
+        end
+
+    end
+
+
+end
+
+function _erf(x::Float32)
+    xabs=abs(x)
+
+  if (xabs< 0.5)
+    # range [0;0.5]
+    # # erf approximation using erf(x)=x+x*P(x^2) with degree 6
+    # Sollya::remez(erf(sqrt(x))/sqrt(x)-1,6,[1e-32;0.5],1,1e-32);
+
+    p=(0.128379167084072442015971708878153077794812875442854468252262977759538466466026108f0 ,-0.37612638677173360887708232154133809982218085424044576090554860522057182310851056f0 ,0.112837843774249243616280397250157777250507394914770984188915122197580134574091983f0 ,-2.68652865375831097248410803484628824158061934615600565461807890879374835870381413f-2 ,5.2188560068141289500478726517878958032487316358404909126782459380536157982980317f-3 ,-8.3948481565340715162646401327604259604058983841328238470236141987304564416187075f-4 )
```

Comment:
these are very long Float32 literals. If you copy and paste them into the repl, Julia will give you minimal length versions.